### PR TITLE
MULE-13255: Invalid versioning for Mule Module Maven Plugin

### PR DESCRIPTION
**Pull request by anosenzo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -243,6 +244,13 @@
     <build>
         <defaultGoal>install</defaultGoal>
 
+        <testResources>
+            <testResource>
+                <filtering>true</filtering>
+                <directory>${basedir}/src/test/projects</directory>
```
Comment (aiannucci) [inline pom.xml]: Why is it called projects instead of resources?

Comment (anosenzo) [inline pom.xml]: I don't know, it was the folder that @pablokraan used it in his tests.

Comment (pablokraan) [inline pom.xml]: resources are accessible on the test's classloader, having them in a separate folder maintains them isolated

Comment (aiannucci) [inline pom.xml]: In that case we need to review the approach in order to not add the projects directory as a test resource.

**Pull request closed without merging**